### PR TITLE
fix: allow sending of messages with empty body content

### PR DESCRIPTION
**Pull request by miaulalala**
Signed-off-by: Anna Larch <anna@nextcloud.com>

### CODE REVIEW

File: lib/Service/MimeMessage.php [hunk 1]
```diff
@@ -127,7 +127,7 @@ public function build(bool $isHtml, string $content, array $attachments): Horde_
 			$bodyPart = new Horde_Mime_Part();
 			$bodyPart->setType('text/plain');
 			$bodyPart->setCharset('UTF-8');
-			$bodyPart->setContents($content);
+			$bodyPart->setContents($content ?? '');
```
Comment (miaulalala) [inline lib/Service/MimeMessage.php:130]: this can go actually

**Review by GretaD — APPROVED** [review]
tested and works <3

**Comment by ChristophWurst** [comment]
Could you extend https://github.com/nextcloud/mail/blob/main/tests/Unit/Service/MimeMessageTest.php to cover the previous error case?

**Review by ChristophWurst — APPROVED** [review]

**Comment by ChristophWurst** [comment]
Does this fix the error `Could not send outbox message 2696: DOMDocument::loadHTML(): Argument #1 ($source) must not be empty` @miaulalala?

Also, what versions are affected and how far do we need to backport?

**Comment by kesselb** [comment]
> Does this fix the error `Could not send outbox message 2696: DOMDocument::loadHTML(): Argument #1 ($source) must not be empty`

Yes :partying_face:

**Pull request merged**